src/drivers/amqp/Queue.php

```diff
@@ -31,6 +31,8 @@ class Queue extends CliQueue
     public $queueName = 'queue';
     public $exchangeName = 'exchange';
     public $vhost = '/';
+    public $heartbeat = 0;
+    public $keepalive = false;
```

Comment:
These are new options. These should be documented and `@since` should be added to their phpdoc.